vdsm_hooks/localdisk/after_disk_prepare

```diff
@@ -180,7 +180,10 @@ def copy_sparse(src_path, dst_path):
 
 def lv_info(lv_name):
     out = helper("list", lv_name)
-    lvs = json.loads(out)["report"][0]["lv"]
+    try:
+        lvs = json.loads(out)["report"][0]["lv"]
+    except:
+        lvs = None
```

Comment:
Well, I was just being pragmatic, I ended up in cases where there was no valid json in out, IIRC because the underlying lvm command did not return anything because the volume was not there.

Reply:
This sounds like a bug in lvm. Do you know how to reproduce it this?

Regarding the way to handle such case, since this is a bug in the code 
we call, failing loudly when the output is not valid json or key is mising
can be good enough, so we may not need to add any error handling here.